Ajoute les notes d'intention des bureaux 2012, 2014 et 2018

MattiSG opened the pull request:
Extrait de la dropbox de l'association.

Fix #7.

@AgileFrance/alumni si vous avez d'autres sources, elles sont bienvenues ! 😃

MattiSG committed:
Crée un historique des notes d'intention

MattiSG committed:
Ajoute la note d'intention du bureau 2018

MattiSG committed:
Ajoute la note d'intention du bureau 2012

MattiSG committed:
Ajoute les URL canoniques des notes d'intention

MattiSG committed:
Ajoute les dates d'élection et le nom d'une liste

MattiSG committed:
Ajuste le balisage pour l'intégration

Le balisage originel était prévu pour des pages seules, pas pour être intégrés dans un autre site

MattiSG committed:
Unifie la présentation des notes d'intention

Unifie les espaces

davidbgk reviewed (approved):
👍

davidbgk commented on _pages/index.md:
```diff
@@ -41,6 +41,17 @@ Le bureau de l'association est actuellement composé de :
+{% assign bureaux = site.bureau | reverse %}
+
+{% for bureau in bureaux %}
+### {{ bureau.election | date: "%Y" }} — [Note d'intention de la liste {{ bureau.liste }}]({{ bureau.url }})
```
Why not a basic list?

MattiSG replied:
Purely presentational (thus bad) reasons. Will fix that  :)

MattiSG committed:
Corrige le balisage

Supprime le trailing whitespace pour assurer un rendu sous la forme d'une <ul><li> correcte
